local.tf

```diff
@@ -32,6 +32,8 @@ locals {
     asg_force_delete              = false                       # Enable forced deletion for the autoscaling group.
     asg_initial_lifecycle_hooks   = []                          # Initital lifecycle hook for the autoscaling group.
     asg_recreate_on_change        = false                       # Recreate the autoscaling group when the Launch Template or Launch Configuration change.
+    default_cooldown              = "300"                       # The amount of time, in seconds, after a scaling activity completes before another scaling activity can start.
```

Comment:
Where did you take the `300`  as default ? Maybe, the default should be `null` ?

Reply:
As i checked the [health_check_grace_period](https://www.terraform.io/docs/providers/aws/r/autoscaling_group.html#health_check_grace_period) probably should be set to `null`,

But the [default_cooldown](https://www.terraform.io/docs/providers/aws/r/autoscaling_group.html#default_cooldown), is not being set by terraform (and should bee 300 as [stated here](https://docs.aws.amazon.com/autoscaling/ec2/userguide/Cooldown.html#cooldown-default)).
That means if set `default_cooldown` to lets say 90 seconds, and later remove this, the value will not get set back to `300` but stay on `90`.

i'm not sure who should enforce defaults (as for example [boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/autoscaling.html#AutoScaling.Client.create_auto_scaling_group) has this default),

But let me know what should be the desired behaviour from this module? null?

Reply:
I would go with null for both as this won't change any behaviour for existing users of the module. I think that is safest option?

Reply:
updated